src/test/java/simplifier/UserControllerIT.java

```diff
@@ -36,6 +35,6 @@ public void createUser() {
        user, User.class);
 
    assertThat(HttpStatus.CREATED, is(responseEntity.getStatusCode()));
-   assertTrue(!responseEntity.hasBody());
+   assertThat(false, is(responseEntity.hasBody()));
```

Comment:
`assertThat(actual, expected)` but not` assertThat(expected, actual)`. Also `assertThat(responseEntity.getBody(), is(nullValue()))` seems more readable